integration-tests/bats/sql-local-remote.bats

```diff
@@ -736,3 +741,168 @@ SQL
 
   [[ "$remoteOutput" == "$localOutput" ]] || false
 }
+
+@test "sql-local-remote: verify dolt tag behavior" {
+  cd altDB
+
+  # get commit hashes
+  headCommit=$(get_commit_hash_at HEAD)
+  secondCommit=$(get_commit_hash_at HEAD~1)
+
+  # show tags
+  run dolt tag
+  [ $status -eq 0 ]
+
+  # add tag without message
+  run dolt tag v1
+  [ $status -eq 0 ]
+
+  # list tags and check new tag is present
+  run dolt tag
+  [ $status -eq 0 ]
+  [[ $output =~ "v1" ]] || false
+
+  # list tags with verbose flag and check new tag is present
+  run dolt tag --verbose
+  [ $status -eq 0 ]
+  [[ $output =~ "v1"$'\t'"$headCommit" ]] || false
+  [[ $output =~ "Tagger: Bats Tests <bats@email.fake>" ]] || false
+
+  # add tag with commit
+  run dolt tag v2 $secondCommit
+  [ $status -eq 0 ]
+
+  # list tags and check new tag is present
+  run dolt tag --verbose
+  [ $status -eq 0 ]
+  [[ $output =~ "v1"$'\t'"$headCommit" ]] || false
+  [[ $output =~ "v2"$'\t'"$secondCommit" ]] || false
+
+  # add tag with message
+  run dolt tag v3 -m "tag message"
+  [ $status -eq 0 ]
+
+  # list tags and check new tag is present
+  run dolt tag --verbose
+  [ $status -eq 0 ]
+  [[ $output =~ "v3"$'\t'"$headCommit" ]] || false
+  [[ $output =~ "tag message" ]] || false
+
+  # add tag with message and commit
+  run dolt tag v4 $secondCommit -m "second message"
+  [ $status -eq 0 ]
+
+  # list tags and check new tag is present
+  run dolt tag --verbose
+  [ $status -eq 0 ]
+  [[ $output =~ "v4"$'\t'"$secondCommit" ]] || false
+  [[ $output =~ "second message" ]] || false
+
+  # add tag with author
+  run dolt tag v5 --author "John Doe <john@doe.com>"
+  [ $status -eq 0 ]
+
+  # list tags and check new tag is present
+  run dolt tag --verbose
+  echo "pavel >>> $output"
+  [ $status -eq 0 ]
+  [[ $output =~ "v5"$'\t'"$headCommit" ]] || false
+  [[ $output =~ "Tagger: John Doe <john@doe.com>" ]] || false
+
+  # delete tag
+  run dolt tag -d v2
+  [ $status -eq 0 ]
+
+  # list tags and check deleted tag is not present
+  run dolt tag --verbose
+  [ $status -eq 0 ]
+  [[ $output =~ "v1"$'\t'"$headCommit" ]] || false
+  [[ ! $output =~ "v2" ]] || false
+  [[ $output =~ "v3"$'\t'"$headCommit" ]] || false
+  [[ $output =~ "v4"$'\t'"$secondCommit" ]] || false
+  [[ $output =~ "tag message" ]] || false
+  [[ $output =~ "second message" ]] || false
+  [[ $output =~ "Tagger: John Doe <john@doe.com>" ]] || false
+
+  cd ../defaultDB
+  start_sql_server defaultDB
+
+  # get commit hashes
+  headCommit=$(get_commit_hash_at HEAD)
+  secondCommit=$(get_commit_hash_at HEAD~1)
+
+  # show tags
+  run dolt tag
```

Comment:
add the --verbose-engine-setup flag, and check the output to confirm we're really going remote. Alter an invocation in the local context as well.

Reply:
Good idead. Adding `--verbose-engine-setup` flag for the first few invocations and verifying that we're getting the results we're expecting.